index.adoc

```diff
@@ -1,17 +1,30 @@
 = WildFly Documentation
+
+This documentation is organized into four main sections:
+
+* *Getting Started with the WildFly Application Server* provides core server information, releases, and APIs.
+* *Build Applications with WildFly* covers using Maven and related development tools.
+* *Build & Deploy WildFly on the Cloud* includes guides for containerization and orchestration.
+* *Provision WildFly & Build Feature Packs* explains advanced provisioning and server customization.
+
```

Comment:
This section appears somewhat oversized in comparison to the rest of the document. Not sure if making it smaller would help. 

From my personal feeling, I think it adds just noise to the landing page and I would not add it at all. What do you think @akil-krsna ?  ... I'm not asking for changes, just would like to get more feedback from others.

Reply:
Ok, let's see if anyone has other feedback and wants to remove them, if not, I'm ok as it is